src/components/VeggyBody/Modal/Modal.jsx

```diff
@@ -0,0 +1,13 @@
+export default function ModalProductItemImage({modalActive, setModalActive, children}) {
```

Comment:
modalActive не використовується 
```suggestion
export default function ModalProductItemImage({ modalActive, setModalActive, children }) {
```